packages/admin-panel-server/src/routes/FetchDataTablePreviewDataRoute.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+import { Request } from 'express';
+
+import { Route } from '@tupaia/server-boilerplate';
+
+export type FetchDataTablePreviewDataRequest = Request<
+  Record<string, never>,
+  Record<string, unknown>,
+  {
+    previewConfig: Record<string, unknown>;
```

Comment:
We should be able to type previewConfig. Add a type in @tupaia/types/src/types/requests/data-table-server/ that can be used by: admin-panel-server (here), data-table-server, and api-client.